javascript/ql/src/semmle/javascript/BasicBlocks.qll

```diff
@@ -67,7 +67,27 @@ private cached module Internal {
   }
 
   cached predicate defAt(BasicBlock bb, int i, Variable v, VarDef d) {
-    v = d.getAVariable() and bbIndex(bb, d, i)
+    exists (VarRef def |
```

Comment:
A `VarRef` called `def` looks like a typo.

Reply:
Yeah, the terminology is not great here. The `VarDef d` is the position in the CFG where the assignment happens, while the `VarRef` is the thing being assigned to. How about renaming it to `lhs`?

Reply:
`lhs` sounds good 👍

Reply:
Renamed.